notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<h2>Batch serve feature values from Vertex AI Feature Store </h2><p> </p><p>You use batch serving to fetch a large number of feature values for high-throughput, such as for model training or batch prediction. In this section, you prepare training examples by using batch serving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>